lib/ExecutionControlEpic/PlanConfigurationFeature/Presenters/PlanConfigurer.js

```diff
@@ -36,6 +36,19 @@ export default class PlanConfigurer
     };
   }
 
+  componentWillReceiveProps(nextProps: Props) {
+    if (
+      this.props.packages.length !== nextProps.packages.length ||
+      this.props.packages !== nextProps.packages
+    ) {
+      this.setState({
```

Comment:
Having new packages doesn't mean it has to be selected, it's only when no package is selected that it should be selected